### DISCUSSION TITLE
Tools and programs for online studying

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#308] [General]
Can you please share your favorites here? Some essential things I should know and have when evaluating my students?

**Comment by XelaAl** [comment]
I want to mention Pear Deck here. Being a teacher, I find it to be very helpful. Your students can draw directly on the presentation, enter their own notes, add numbers, drag and drop symbols, or answer multiple-choice questions. It has a friendly integration with Google Slides, too.

**Comment by hernessteve** [comment]
Did you hear about proctoedu [https://proctoredu.com/](https://proctoredu.com/) Great tool which every single teacher should use if he/she values the reputation. It does the whole process of verification students identity during online assessments and exams.
